tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+# Step by step tutorials
+
+## Setup
+
+For running the examples in the tutorials you can either manually install Sockeye, it's dependencies and additional
```

Comment:
"[...] either install Sockeye manually and its dependencies and additional tools, or use the Dockerfile provided which comes with everything included."